webrtc.html

```diff
@@ -509,33 +509,47 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>If <var>connection</var>'s <a href=
+              "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+              signalingState</a> is <code>closed</code>, throw an
+              <code>InvalidStateError</code> exception and abort these steps.</p>
+          </li>
+
+          <li>
+            <p>If <code>operations</code> is empty, execute the
+              function and let <var>p</var> be the promise returned by
+              the function.  If <code>operations</code> is non-empty,
+              let <var>p</var> be a promise equivalent to the promise
+              that would be returned by calling the function (it will
+              be settled at the same time and with the same result).
```

Comment:
Great, looking forward to it. My point about "equivalent" and "settled at the same time", was that they seem like unnecessary requirements given the problem space. Promises have well-defined semantics already (even in c++ implementations), so we should be able to express our needs using them, without inventing new semantics.